src/reducers/files-and-folders-metadata/files-and-folders-metadata-types.ts

```diff
@@ -0,0 +1,23 @@
+export const INIT_FILES_AND_FOLDERS_METADATA = "FF_METADATA/INIT";
```

Comment:
Pour être consistant avec `reducers/files-and-folders/files-and-folders-types.ts`, la string devrait être `"FILES_AND_FOLDERS_METADATA/INIT"`